Source/ACE.Common/ConfigManager.cs

```diff
@@ -19,13 +19,29 @@ public static void Initialize(MasterConfiguration configuration)
         }
 
         /// <summary>
-        /// initializes from a config.json file specified by the path
+        /// initializes from a Config.js file specified by the path
         /// </summary>
-        public static void Initialize(string path = @"Config.json")
+        public static void Initialize(string path = @"config.js")
         {
+            string fpOld = Path.Combine(Environment.CurrentDirectory, path);
+            string fpNew = Path.Combine(Environment.CurrentDirectory, Path.GetFileNameWithoutExtension(path) + ".js");
+            string fpChoice = null;
             try
             {
-                Config = JsonConvert.DeserializeObject<MasterConfiguration>(new JsMinifier().Minify(File.ReadAllText(path)));
+                if (!File.Exists(fpNew) && File.Exists(fpOld))
```

Comment:
This should be unneeded. The code for this was posted yesterday:

if not exist "$(ProjectDir)Config.js" if exist "$(ProjectDir)Config.json" move "$(ProjectDir)Config.json" "$(ProjectDir)Config.js"

Reply:
there are situations where this is needed

Reply:
It still seems like this shouldn't be needed in the code to me, but since it can catch some possible edge cases at runtime, and shouldn't affect anything besides those special cases, I guess it should be fine.

Just trying to keep the core code as clean as possible, and hopefully after everyone has the new config filename, this code can eventually be removed

Reply:
That's reasonable.  Let's plan on removing the runtime patcher in say, 6 months.